examples/count_resources.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import urllib3
+import argparse
+
+# ==========================
+# 🔧 CONFIGURATION
+# ==========================
+API_HOST_URL = 'https://elab.ub.uni-frankfurt.de/api/v2'  # <-- URL eLabFTW Goethe Uni or Replace with your instance URL 
```

Comment:
all examples must work out of the box with the dev instance: so as an example, use https://elab.local:3148 and key `apiKey4Test`.